src/metabase/util/encryption.clj

```diff
@@ -147,22 +147,22 @@
   "If `MB_ENCRYPTION_SECRET_KEY` is set and `v` is encrypted, decrypt `v`; otherwise return `s` as-is. Attempts to check
   whether `v` is an encrypted String, in which case the decrypted String is returned, or whether `v` is encrypted bytes,
   in which case the decrypted bytes are returned."
-  {:arglists '([secret-key? s & {:keys [log-errors?], :or {log-errors? true}}])}
+  {:arglists '([secret-key? s & [{:keys [log-errors?], :or {log-errors? true}}]])}
```

Comment:
Why was this changed? It doesn't look like `log-errors?` is used by callers anywhere so we should be able to remove this argument anyway.

Reply:
Hmmm, good question, I'm going to remove that.